dwds/lib/src/debugging/instance.dart

```diff
@@ -354,8 +354,8 @@ class InstanceHelper extends Domain {
         count: count, elementCount: elements.length, length: length);
     final range = elements.sublist(0, rangeCount);
 
-    return Future.wait(range
-        .map((element) async => await _instanceRefForRemote(element.value)));
+    return Future.wait(
+        range.map((element) async => _instanceRefForRemote(element.value)));
```

Comment:
```suggestion
        range.map((element) => _instanceRefForRemote(element.value)));
```

Less interesting case since it doesn't become a tearoff, but if the sole `await` gets removed it is worth dropping the `async` too.

Reply:
Thanks Nate, I will skip this PR for now and add the suggested change separately.